docs/manual/cli.qmd

```diff
@@ -0,0 +1,151 @@
+```{python}
+#| echo: false
+from odm_sharing.tools.share import OutFmt, share
+
+from common import DATA, SCHEMA, load_csv_md, print_file
+```
+
+# CLI {#sec-cli}
+
+## Reference
+
+```bash
+odm-share [OPTION]... SCHEMA INPUT
+```
+
+Arguments:
+
+- SCHEMA
+
+  sharing schema file path
+
+- INPUT
+
+  spreadsheet file path or [SQLAlchemy database URL](https://docs.sqlalchemy.org/en/20/core/engines.html#database-urls)
+
+Options:
+
+- `--orgs=NAME[,...]`
+
+    comma separated list of organizations to output data for, defaults to all
+
+- `--outfmt=FORMAT`
+
+    output format (excel or csv), defaults to input format
+
+- `--outdir=PATH`
+
+    output file directory, defaults to the current directory
+
+- `-d`, `--debug`:
+
+    output debug info to STDOUT (and ./debug.txt) instead of creating sharable
+    output files. This shows which tables and columns are selected, and how
+    many rows each filter returns.
+
+- `-q`, `--quiet`:
+
+    don't log to STDOUT
+
+One or multiple sharable output files will be created in the chosen output
+directory according to the chosen output format and organization(s). Each
+output file will have the input name followed by the recipient org name.
+
+### Errors
+
+Error messages will be printed to the terminal (STDERR) when something is
+wrong. The message starts with telling where the error originated, including
+the filename and line number or rule id. Here's a few examples:
+
+When headers are missing from the schema:
+
+```{python}
+#| echo: false
+share('assets/schema-missing-headers.csv', 'assets/measures.csv')
+```
+
+When no share-rules are contained in the schema:
+
+```{python}
+#| echo: false
+share('assets/schema-missing-rules.csv', 'assets/measures.csv')
+```
+
+## Usage
+
+### Examples
```

Comment:
I think these examples should be more fully specified with example input and schema files, and showing what the output looks like. This way the user can run the example on the computer to see if they get the same results.

Reply:
I would also expand the titles more? For example, using a CSV file as an input.

Reply:
done